test/udp_sink_test.rb

```diff
@@ -58,6 +58,8 @@ def test_socket_error_should_invalidate_socket
     udp_sink = StatsD::Instrument::UDPSink.new('localhost', 8125)
     udp_sink << 'foo:1|c'
     udp_sink << 'bar:1|c'
+    # Let the dispatcher thread emit
+    sleep 0.1
```

Comment:
would it be nice to have a `udp_sunk.flush` or another method that blocks until the queue is empty?